app/MMB/src/components/menu/MenuGroup.js

```diff
@@ -2,13 +2,12 @@
 
 import * as React from 'react';
 import { View, Platform } from 'react-native';
-import { SimpleCard } from '@kiwicom/mobile-shared';
 
 import { SeparatorFullWidth, SeparatorTrimmed } from '../Separators';
 
 type Props = {|
   children: React.Node,
-  withoutIcons: boolean,
```

Comment:
Did we remove this during our refactoring? It's wrong - the separators are not correct in the "Help" for example (they should be short).

Reply:
Yes you did, I will check the "Help"

Reply:
Added `short` custom separator to help and other menu items